src/Bridge/Doctrine/Orm/Extension/QueryResultItemExtensionInterface.php

```diff
@@ -33,8 +33,10 @@ public function supportsResult(string $resourceClass, string $operationName = nu
 
     /**
      * @param QueryBuilder $queryBuilder
+     * @param string       $resourceClass
+     * @param string|null  $operationName
      *
      * @return mixed
      */
-    public function getResult(QueryBuilder $queryBuilder);
+    public function getResult(QueryBuilder $queryBuilder/*, string $resourceClass, string $operationName = null*/);
```

Comment:
To keep consistency with `QueryResultCollectionExtensionInterface` but yes, maybe it's useless...